project/WithDebugCommand.scala

```diff
@@ -19,7 +19,7 @@ object WithDebugCommand {
     "-agentlib:jdwp=transport=dt_socket,server=n,address=localhost:5005,suspend=y";
 
   val truffleNoBackgroundCompilationOptions = Seq(
-    "-Dpolyglot.engine.BackgroundCompilation=false"
```

Comment:
Revert, this will be fixed in https://github.com/enso-org/enso/pull/8467

Reply:
Done in 8fda9f5700